src/plot/sample/quality_score_distribution.rs

```diff
@@ -81,8 +85,12 @@ impl SamplePlot for QualityScoreDistributionPlot {
         plot.add_trace(trace);
 
         // (5) Configure the graph for plotting and return.
+        let title = match sample {
+            Some(s) => format!("{} - {}", self.name(), s.clone()),
+            None => self.name().to_string(),
+        };
         let layout = Layout::new()
-            .title(Title::new(self.name()))
+            .title(Title::new(title.as_str()))
```

Comment:
```suggestion
            .title(Title::new(&title))
```

Reply:
`String` implements [`AsRef<str>`](https://doc.rust-lang.org/std/string/struct.String.html#impl-AsRef%3Cstr%3E-for-String), so you can generally do this a little bit cleaner.